message/validation/validation.go

```diff
@@ -180,15 +200,23 @@ func (mv *messageValidator) obtainValidationLock(messageID spectypes.MessageID)
 	return mutex
 }
```

Comment:
are we planning to really lock by msgid eventually? here there's just a global block, at least lets add a todo or some description.

Reply:
@y0sher im seeing here a lock per msgid, not sure if that was changed since you last checked?

Reply:
@y0sher we use the global lock only for access to map with locks by message ID, I'm not sure I understand what you mean